### PR TITLE
Added dryml example for Test Case: Filter Coffee

### PR DESCRIPTION
**Pull request by zuck3rman**
Signed-off-by: Andrew Zuck3rman <andrew.sakharoff@gmail.com>

### CODE REVIEW

File: DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        	temperature_level = hot
+        	volume = 0.1[liter]
+        pourover
+        	temp_level = cold
+        filter
+        	colour = white
+        	material = paper
+        	temp_level = cold
+	out:
+		pourover
```
Comment (vgrigoryevskiy) [inline DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml:86]:
```suggestion
	pourover
```

File: DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+	parameters:
+		weight
+
+water
+	states:
+		normal
+		boiled
+	parameters:
+		temperature_level
+		volume
```
[the remark this answers is not in the document]
Comment (zuck3rman) [reply DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml:24]: temperature_level is decided to be used. filter_coffee.dryml updated accordingly.

File: DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+filter
+	parameters:
+		colour
+		material
+		temp_level
```
[the remark this answers is not in the document]
Comment (zuck3rman) [reply DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml:30]: temperature_level is decided to be used. filter_coffee.dryml updated accordingly.

File: DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml (hunk 2 above)
[the remark this answers is not in the document]
Comment (zuck3rman) [reply DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml:24]: volume has been removed from the parameters

File: DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml (hunk 3 above)
[the remark this answers is not in the document]
Comment (zuck3rman) [reply DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml:30]: volume has been removed from the parameters

File: DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+cook_top
+	states:
+		functioning
+
+filtered_coffee
+	states:
+		liquid
+	parameters:
+		volume
```
[the remark this answers is not in the document]
Comment (zuck3rman) [reply DOCUMENTATION/Test Cases/filter_coffee/filter_coffee.dryml:52]: Done

**Pull request merged**